api/src/pcapi/templates/extracts/beneficiary_extract.html

```diff
@@ -0,0 +1,204 @@
+                </table>
+            {% endfor %}
+        {% endif %}
+
+        {% if container.internal.actionsHistory %}
+            <h3>⛔ <span class=underline>Historique des blocages du compte « pass Culture »</span></h3>
+            {% for action in container.internal.actionsHistory %}
+                <table class="borderless">
+                    <tr><td>Date</td><td>{{action.actionDate.strftime('%d/%m/%Y %H:%M:%S')}}</td></tr>
+                    <tr><td>Action</td><td>{{action.actionType.value}}</td></tr>
+                </table>
+            {% endfor %}
+        {% endif %}
+
+        {% if container.internal.beneficiaryValidations %}
+            <h3>👌 <span class=underline>Validations d'identité</span></h3>
+            {% for validation in container.internal.beneficiaryValidations %}
+                <table class="borderless">
+                    <tr><td>Date de la validation</td><td>{{validation.dateCreated.strftime('%d/%m/%Y %H:%M:%S')}}</td></tr>
+                    <tr><td>Moyen de la validation</td><td>{{validation.type.value}}</td></tr>
+                    {% if validation.status %}<tr><td>Résultat</td><td>{{validation.status.value}}</td></tr>{%endif%}
+                    {% if validation.updateAt %} <tr><td>Dernière modification</td><td>{{validation.updatedAt.strftime('%d/%m/%Y %H:%M:%S')}}</td></tr>{%endif%}
+                </table>
+            {% endfor %}
+        {% endif %}
+
+        {% if container.internal.deposits %}
+            <h3>💰 <span class=underline>Crédits</span></h3>
+            {% for deposit in container.internal.deposits %}
+                <table class="borderless">
+                    <tr><td>Date d'obtention</td><td>{{deposit.dateCreated.strftime('%d/%m/%Y %H:%M:%S')}}</td></tr>
+                    {% if deposit.expirationDate %}<tr><td>Date d'expiration</td><td>{{deposit.expirationDate.strftime('%d/%m/%Y %H:%M:%S')}}</td></tr>{% endif %}
+                    <tr><td>Valeur</td><td>{{('%0.2f' % deposit.amount|float) | replace('.', ',')}}€</td></tr>
+                    <tr><td>Source</td><td>{{deposit.source}}</td></tr>
+                    {% if deposit.dateUpdated %}<tr><td>Dernière modification</td><td>{{deposit.dateUpdated.strftime('%d/%m/%Y %H:%M:%S')}}</td></tr>{% endif %}
+                    <tr><td>Type de crédit</td><td>{{deposit.type.value}}</td></tr>
+                </table>
+            {% endfor %}
+        {% endif %}
+
+        {% if container.internal.bookings %}
+            <h3>📅 <span class=underline>Réservations effectuées depuis l’application « pass Culture »</span></h3>
+            {% for booking in container.internal.bookings %}
+                <table class="borderless">
+                    <tr><td>Nom</td><td>{{booking.name}}</td></tr>
+                    <tr><td>Quantité</td><td>{{booking.quantity}}</td></tr>
+                    <tr><td>Prix unitaire</td><td>{{'%0.2f' % booking.amount|float }}€</td></tr>
```

Comment:
```suggestion
                    <tr><td>Prix unitaire</td><td>{{ ('%0.2f' % deposit.amount|float) | replace('.', ',') }} €</td></tr>
```
![image](https://github.com/pass-culture/pass-culture-main/assets/23212130/6288d240-63b9-41eb-a4e7-1d5342aa8e27)